### ISSUE TITLE
[윈도우 모니터링] 기술 스택 선정

### ISSUE THREAD

**Issue by tea7day** [echoen/enjoy-build#33]

**Comment by TRQ1** [comment]
EC2 완성후 컨테이너 환경으로 변경은 어때요?
Undertow Docker화는 되어있는걸로 알고있습니다.

**Comment by tea7day** [comment]
좋네 도커기반으로 하는데 한표~

**Comment by tea7day** [comment]
위에 기술 스택 정리해놨습니다~
부족한 부분은 채워주시고! 자기가 하고싶은 부분에 이름을 적으면 될 것 같습니다ㅎㅎ
단, 이왕이면 한 파트당 2명씩 되면 좋을 것 같습니다. (강제는 아님!)

**Comment by TRQ1** [comment]
아니요 생각해보니.. 제가 도커라이즈하네요 이건 제가 따로 진행하겠습니다. 아무래도 배포 환경때문에 도커화가 좋을것 같습니다.

**Comment by tea7day** [comment]
undertow는 스프링부트에 내장되기 때문에 별도의 was 서버는 필요하지 않아서 제외했습니다~
대신 스프링부트 도커라이징이 필요할듯요!
